### PR TITLE
Alpha to dev

### PR DESCRIPTION
**Pull request by rikukissa**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

**Review by jamil314 — APPROVED** [review]

File: src/translations/client.csv [hunk 1]
```diff
@@ -2601,6 +2605,7 @@ v2.event.birth.action.certificate.form.section.collectPayment.service.label.befo
 v2.event.birth.action.certificate.form.section.collectPayment.service.label.inBetweenRegistrationTargets,Birth registration after 30 days but before 365 days of date of birth,Birth registration after 30 days but before 365 days of date of birth,Enregistrement de naissance après 30 jours mais avant 365 jours de la date de naissance
 v2.event.birth.action.collect-certificate.label,This is shown as the action name anywhere the user can trigger the action from,Print,Imprimer
 v2.event.birth.action.correct.label,This is shown as the action name anywhere the user can trigger the action from,Correct record,Enregistrement correct
+v2.event.birth.action.correction.form.section.requester.identity.verify.title,This is the title of the section,Verify their identify,Vérifier l'identité
```
Comment (Nil20) [inline src/translations/client.csv:2607]: We've changed this label to `Verify ID`, you can see it here in [develop](https://github.com/opencrvs/opencrvs-countryconfig/blob/develop/src/translations/client.csv#L2596)

Comment (Nil20) [reply src/translations/client.csv:2607]: https://github.com/opencrvs/opencrvs-farajaland/pull/1609
This PR should handle the change for e2e test

File: src/form/v2/birth/forms/correctionForm/index.ts [hunk 2]
```diff
@@ -121,7 +121,7 @@ export const CORRECTION_FORM = defineActionForm({
       type: PageTypes.enum.VERIFICATION,
       title: {
         id: 'v2.event.birth.action.correction.form.section.requester.identity.verify.title',
-        defaultMessage: 'Verify their identity',
+        defaultMessage: 'Verify ID',
```
Comment (Nil20) [inline src/form/v2/birth/forms/correctionForm/index.ts]: This one, should be updated in `client.csv`

**Review by tareq89 — APPROVED** [review]
Neat

File: src/api/certificates/handler.ts [hunk 3]
```diff
@@ -34,6 +38,12 @@ export interface ICertificateConfigData {
   }
   svgUrl: string
   fonts?: Record<string, FontFamilyTypes>
+  conditionals?:
+    | {
+        type: 'SHOW'
+        conditional: JSONSchema
+      }[]
+    | undefined
```
Comment (Nil20) [inline src/api/certificates/handler.ts]: Isn't `| undefined` being redundant?

**Review by Nil20 — APPROVED** [review]
